client/state/document/reducer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import filter from 'lodash/filter';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DOCUMENT_LINK_ADD,
+	DOCUMENT_LINK_REMOVE,
+	DOCUMENT_META_ADD,
+	DOCUMENT_META_REMOVE,
+	DOCUMENT_DESCRIPTION_SET,
+	DOCUMENT_TITLE_SET,
+	DOCUMENT_UNREAD_COUNT_SET
+} from 'state/action-types';
+
+export function title( state = '', action ) {
+	switch ( action.type ) {
+		case DOCUMENT_TITLE_SET:
+			return action.title;
+	}
+
+	return state;
+}
+
+export function unread_count( state = 0, action ) {
```

Comment:
Naming conventions dictate camelCase, not snake_case ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions))